### PR TITLE
pagination - create empty file

### PR DESCRIPTION
**Pull request by markosolopenko**

### CODE REVIEW

File: src/common/components/Pagination/Pagination.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import { ReactComponent as ArrowLeft } from '../../../assets/arrowLeft.svg';
+import { ReactComponent as ArrowRight } from '../../../assets/arrowRight.svg';
+
+import s from './Pagination.module.scss';
+
+export const Pagination = ({
+  onChange,
+  activePage,
+  totalPages,
+  arrowForwardOnClick,
```
Comment (apostoliuk97) [inline src/common/components/Pagination/Pagination.js]: use onChange for buttons too

File: src/common/components/Pagination/Pagination.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  arrowBackOnClick,
+}) => {
+  const [page, setValue] = useState(activePage);
```
Comment (apostoliuk97) [inline src/common/components/Pagination/Pagination.js]: `setPage`

File: src/common/components/Pagination/Pagination.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  useEffect(() => {
+    setValue(activePage);
+  }, [activePage]);
+  console.log(page);
```
Comment (apostoliuk97) [inline src/common/components/Pagination/Pagination.js]: remove logger

File: src/common/components/Pagination/Pagination.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+export const Pagination = ({
+  onChange,
+  activePage,
+  totalPages,
+  arrowForwardOnClick,
+  arrowBackOnClick,
+}) => {
+  const [page, setValue] = useState(activePage);
+  useEffect(() => {
+    setValue(activePage);
```
Comment (apostoliuk97) [inline src/common/components/Pagination/Pagination.js]: check it out, do you need the local state for pagination component

File: src/common/components/Pagination/Pagination.js (hunk 1 above)
Comment (markosolopenko) [reply src/common/components/Pagination/Pagination.js]: done

File: src/common/components/Pagination/Pagination.js (hunk 2 above)
Comment (markosolopenko) [reply src/common/components/Pagination/Pagination.js]: deleted unneeded useState

File: src/common/components/Pagination/Pagination.js (hunk 4 above)
Comment (markosolopenko) [reply src/common/components/Pagination/Pagination.js]: no it wasn't needed so I deleted it

File: src/common/components/Pagination/Pagination.js (hunk 3 above)
Comment (markosolopenko) [reply src/common/components/Pagination/Pagination.js]: done

**Pull request merged**